libidn2/plan.sh

```diff
@@ -0,0 +1,30 @@
+pkg_name=libidn2
+pkg_origin=core
+pkg_version="2.0.4"
+pkg_description="..."
```

Comment:
I'd personally recommend either getting rid of this or adding a quick description. The libidn package doesn't even have one, so I'm cool with it being removed.